cl/opinion_page/forms.py

```diff
@@ -237,6 +258,70 @@ class CourtUploadForm(forms.Form):
             }
         ),
     )
+
+    disposition = forms.CharField(
+        label="Disposition",
+        required=False,
+        widget=forms.Textarea(
+            attrs={
+                "class": "form-control",
+            }
+        ),
+    )
+
+    summary = forms.CharField(
+        label="Summary",
+        required=False,
+        widget=forms.Textarea(attrs={"class": "form-control"}),
+    )
+
+    # TODO this could be a select field with hardcoded options for a specific court
```

Comment:
?

Reply:
I thought that if we have the lower court names, we could create a list with lower court names, so that the court names are normalized